lib/view_model/students.dart

```diff
@@ -132,6 +132,10 @@ class StudentsLogic extends ChangeNotifier {
     _selectedStudents.clear();
   }
 
+  specificStudentsCleared(){
```

Comment:
You do not need to create a function for this, since the code is readable on its own and the function is only one line.

specificStudentsCleared is not easier to read or more expressive than _specificStudents.clear()

Reply:
solved